app/models/github/team_member.rb

```diff
@@ -1,8 +1,6 @@
 class Github::TeamMember < ApplicationRecord
-  has_one :user,
-    foreign_key: :uid,
-    primary_key: :user_id,
-    class_name: "User",
+  belongs_to :user,
+    primary_key: :uid,
     inverse_of: :github_team_memberships,
-    dependent: :destroy
+    optional: true
```

Comment:
There is currently a [not null constraint on the database](https://github.com/exercism/website/blob/main/db/schema.rb#L529), so this isn't actually optional. So either the DB or this line should be changed.

Reply:
I've changed things a bit, but the factory is not setup well enough apparently:

> ActiveRecord::RecordInvalid: Validation failed: User must exist

Any ideas?

Reply:
This is green now, so I guess this is an old comment?